<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Enter</class>
 <widget class="QMainWindow" name="Enter">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>577</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>449</width>
    <height>577</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>449</width>
    <height>577</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#608d8c</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>451</width>
      <height>491</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>451</width>
      <height>491</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>451</width>
      <height>491</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #405e5e</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="enter">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>ВХОД</string>
     </property>
    </widget>
    <widget class="QLabel" name="login">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ЛОГИН</string>
     </property>
    </widget>
    <widget class="QLabel" name="password">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>ПАРОЛЬ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inp_log">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border: 2px solid;
border-width: 2px;
border-radius: 15px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="inp_passw">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid;
border-width: 2px;
border-radius: 15px;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ent">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>310</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myanmar Text</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(145, 182, 182);
	border-width: 2px;
	border-radius: 15px;
}

QPushbutton:pressed {
	background-color: white

}</string>
     </property>
     <property name="text">
      <string>войти</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>ПОМОЩНИК ПРИ ВЫБОРЕ ПРОФЕССИИ</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="D:/prof.qrc"/>
 </resources>
 <connections/>
</ui>
